src/main/java/use_cases/complete_task/CompleteTaskPresenter.java

```diff
@@ -18,8 +28,18 @@ public class CompleteTaskPresenter implements CompleteTaskOutputBoundary {
      */
     @Override
     public String taskCompleted(CompletedTaskModel taskCompleted) {
-        return "The task " + taskCompleted.getTaskName() + " from " +
```

Comment:
Excellent change here! I think that having our presenters depend on a view interface allows us to more strictly adhere to SRP while also using dependency inversion; this will definitely help us in maintain clean architecture principles